draft-nygren-httpbis-httpssvc.md

```diff
@@ -574,6 +517,41 @@ Authoritative DNS servers SHOULD return A, AAAA, and HTTPSSVC records
 any in-bailiwick SvcDomainNames.
 
 
+# Performance optimizations
+
+For optimal performance (i.e. minimum connection setup time), clients
+SHOULD issue address (AAAA and/or A) and HTTPSSVC queries
+simultaneously, and SHOULD implement a client-side DNS cache.
+With these optimizations in place, and conforming DNS servers,
+using HTTPSSVC does not add network latency to connection setup.
+
+Using HTTPSSVC with HTTP/3 saves a roundtrip during connection setup,
+compared to HTTP over TLS 1.3 over TCP.  Performance-sensitive
+server operators SHOULD offer HTTP/3 in their HTTPSSVC records.
+
```

Comment:
Does it?  I'd think with the source-address-token-validation it's the same number of RTTs for a cold client as the TCP+TLS 1.3 1RTT flow?  (Perhaps I'm misremembering.)

Reply:
That's true, but I thought most servers were expected not to perform that kind of validation unless an attack is detected.